lib/static/modules/middlewares/metrika.js

```diff
@@ -5,12 +5,19 @@ let metrika;
 
 export default metrikaClass => store => next => action => {
     switch (action.type) {
-        case actionNames.INIT_GUI_REPORT: {
+        case actionNames.INIT_GUI_REPORT:
+        case actionNames.INIT_STATIC_REPORT: {
+            const startLoadTime = Date.now();
+
             const result = next(action);
-            const config = get(store.getState(), 'config.yandexMetrika', {});
+            const state = store.getState();
+            const config = get(state, 'config.yandexMetrika', {});
+            const testsCount = get(state, 'tree.browsers.allIds.length', 0);
 
             metrika = metrikaClass.create(config);
 
+            metrika.sendVisitParams({tti: Date.now() - startLoadTime, testsCount});
```

Comment:
I send to ym not only tti in ms, but also tests count which helps to determine why tti is so slow or so fast.